gpu_bdb/queries/q26/gpu_bdb_query_26_dask_sql.py

```diff
@@ -46,10 +50,16 @@ def get_clusters(client, kmeans_input_df):
     results_dict = client.compute(*ml_tasks, sync=True)
 
     output = kmeans_input_df.index.to_frame().reset_index(drop=True)
-
-    labels_final = dask_cudf.from_cudf(
-        results_dict["cid_labels"], npartitions=output.npartitions
-    )
+    
```

Comment:
Any reason we can't use the same `get_clusters` for both the dask_cudf and dask-sql versions of the query like in query 20?